### ISSUE TITLE
False error "exceeded user lifetime quota"

### ISSUE THREAD

**Issue by tomsturges** [AwesomeTTS/awesometts-anki-addon#254]
I purchased AwesomeTTS pro (small) a couple days ago. I am getting a false error about exceeding lifetime quota. The advertised quota is 250,000 characters. I have added audio to 836 cards (German language). I exactly counted the number of characters by exporting everything to a .csv file and using a word counter. There are precisely 12,667 characters (about 15 characters per card which makes sense as this is just single (or two) word vocabulary). 

The following problem was encountered:
Status code: 429 (b'{"error": "Exceeded User lifetime quota)"}\n') (14 times)

Please help

**Comment by luc-languagetools** [comment]
Hi, you should have received a new API key after subscribing to the paid plan, and using that API key you shouldn't get the error message above. Please contact me languagetools@mailc.net if you need more details.

**Comment by tomsturges** [comment]
Ah apologies. It's weird though. I signed up for the trial the day before and never received an API so the next day I thought oh well let's just buy a subscription. Then I received two emails at exactly the same time. Now looking more closely the first one was the trial key and the second one was the paid plan key. That's why I got confused. But weird that the trial email only arrived the next day at exactly the same time as the paid plan key (which arrived within seconds of paying). I dunno, maybe something for you to look into, just letting you know :)
